Samples/Server/Unity-FractalStream/Assets/WebRTCServer.cs

```diff
@@ -169,4 +179,9 @@ void OnInputData(string val)
                 break;
         }
     }
+
+	void OnLog(int level, string message)
+	{
+		Debug.Log(string.Format("[{0}] : {1}", level, message));

```

Comment:
I suggest that we only output the debug log in the development build so the above code should be put under "if (Debug.isDebugBuild)". More info here: https://docs.unity3d.com/ScriptReference/Debug-isDebugBuild.html.